climakitae/explore2.py

```diff
@@ -1,21 +1,23 @@
 # A class for holding the app explore options
 
 import panel as pn
+import param
+from .tmy import AverageMeteorologicalYear, _amy_visualize
+
+#-----------------------------------------------------------------------
 
 class AppExplore(object):
     """
     A class for holding the following app explore options:
-        app.explore2.TMY()
+        app.explore2.amy()
         app.explore2.thresholds()
     """
 
```

Comment:
You and @ckingdon95 both have different versions of this same file. I suggest you communicate between each other about how to best combine these, because we will have merge conflicts with this file.

Reply:
RE: this, since "TMY" was the old version... i'm wondering if the _order_ in which we merge this might resolve this? As in Cora merges #60 first, and then I merge #85 because mine will update it with the correct "amy" phrasing? Could also just be wilding misunderstanding/misrepresenting how github merges work too

Reply:
@vicford my understanding is that there will be merge conflicts regardless of the who merges first. But, it should be relatively easy to resolve the conflict. Perhaps @ckingdon95 (resident GitHub whiz) can take the lead on resolving the conflict when she merges her branch (i.e. if you are ready to merge yours now before her, I think thats fine).